zntrack/dvc/__init__.py

```diff
@@ -32,40 +32,62 @@
 
 
```

Comment:
For the the dvc module they are a collection of all available dvc run options.
I'll update the module docstring and reference
https://github.com/zincware/ZnTrack/blob/a5868a4987ea20b00e52f21e6724b26d16fe7519/zntrack/dvc/__init__.py#L30-L31

They are mostly indistinguishable except for their name and could be considered structural data.
I prefer the reference to the official DVC documentation here. They are all lowercase, because the commands in DVC are also lowercase. I think  `zn.outs()` looks better than `zn.Outs()` (similar to `dvc run --outs <file>`).

For the `zntrack.zn` this is pretty much the same https://github.com/zincware/ZnTrack/blob/a5868a4987ea20b00e52f21e6724b26d16fe7519/zntrack/zn/__init__.py#L34-L37
except that the `__init__` is a lot more crowded and I will put everything that is more complex than
https://github.com/zincware/ZnTrack/blob/a5868a4987ea20b00e52f21e6724b26d16fe7519/zntrack/zn/__init__.py#L155-L156
into a seperate file eventually.

I honestly don't know what to put into  the docstrings that would not fit better into the module docstring.

Reply:
I can understand referencing the DVC docs but in general it makes trying to work with any of the methods difficult. If you will only refer to dvc docs perhaps at least link that in the doc string so that there is a clear path to the reference. If someone jumps with their IDE to the method it will land at that method not necessarily at the top of the file. At least even an overview of why they are different. They all have different purposes and uses and so the uniformity requirement of doc-strings standards is not really met.

For the naming, I am against violating naming conventions as they are very helpful and a convention for a reason. At the end of the day that is your choice.

Reply:
>  If you will only refer to dvc docs perhaps at least link that in the doc string so that there is a clear path to the reference
I will do that.

Reply:
That is your decision.